### DISCUSSION TITLE
Idea: spacer widget draws a line

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#2854] [Ideas]
widget.Spacer can i make a line instead of spacer?
![2021-10-07-022627_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/136297140-b1999f63-77e5-44ad-bffd-209d1dac7274.png)

**Comment by elParaguayo** [comment]
Thanks for the suggestion.

It's not clear from your screenshot exactly what you're asking. Do you want a horizontal line filling a space or a vertical line between widgets? If it's the latter, `widget.Sep` should already do that.

Regardless, if #2489 gets merged, you'll be able to do both.

**Reply by ghost** [reply]
yes to fill in the blank 
my screen is 1280x1024 but i change it to higher
more spaces are formed, it is necessary to make a character creation to fill it
